backend/core/src/feed.rs

```diff
@@ -13,10 +13,23 @@ pub mod connector;
 
 use connector::Serialized;
 
-pub trait FeedMessage: Serialize {
+pub trait FeedMessage {
     const ACTION: u8;
 }
 
+pub trait FeedMessageWrite: FeedMessage {
```

Comment:
Missing docs (which seems to be a pretty common thing in the project in general).